### PR TITLE
Fix the sorting order

### PR DESCRIPTION
**Pull request by Vifon**
The ascending/descending orders seemed to be reversed.

### CODE REVIEW

File: src/elisp/treemacs-customization.el [hunk 1]
```diff
@@ -209,7 +209,7 @@ To keep the alist clean changes should not be made directly, but with
   :type 'regexp
   :group 'treemacs)
 
-(defcustom treemacs-sorting 'alphabetic-desc
+(defcustom treemacs-sorting 'alphabetic-asc
```
Comment (Alexander-Miller) [inline src/elisp/treemacs-customization.el]: Installation snippet in the readme should be updated as well.

File: src/elisp/treemacs-rendering.el [hunk 2]
```diff
@@ -97,53 +97,53 @@ the height of treemacs' icons must be taken into account."
   "Sort F1 and F2 alphabetically asc."
   (declare (pure t) (side-effect-free t))
   (inline-letevals (f1 f2)
-    (inline-quote (string-lessp ,f2 ,f1))))
```
Comment (Alexander-Miller) [inline src/elisp/treemacs-rendering.el]: Commits should be squashed, commit message should be prefixed with `[Rendering]`.

File: src/elisp/treemacs-rendering.el [hunk 3]
```diff
@@ -97,53 +97,53 @@ the height of treemacs' icons must be taken into account."
...
+    (inline-quote (string-lessp ,f1 ,f2))))
```
Comment (Alexander-Miller) [inline src/elisp/treemacs-rendering.el]: Is `make test` still green?

File: src/elisp/treemacs-customization.el (hunk 1 above)
Comment (Vifon) [reply src/elisp/treemacs-customization.el]: Done.

File: src/elisp/treemacs-rendering.el (hunk 2 above)
Comment (Vifon) [reply src/elisp/treemacs-rendering.el]: Done.

File: src/elisp/treemacs-rendering.el (hunk 3 above)
Comment (Vifon) [reply src/elisp/treemacs-rendering.el]: Yes, it is.

**Comment by Alexander-Miller** [comment]
:+1:

**Pull request merged**